### PR TITLE
#486: Unify the loader

### PR DESCRIPTION
**Pull request by hani-ibrahim**
Working on #486

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/LoaderView.swift [hunk 1]
```diff
@@ -27,6 +28,14 @@ class LoaderView: UIView {
         hidden = true
     }
 
+    internal static func displayLoader(block: (() -> Void) -> Void) {
```
Comment (bonkey) [inline AtlasUI/AtlasUI/LoaderView.swift]: ihwcaj6r061wc

Comment (bonkey) [inline AtlasUI/AtlasUI/LoaderView.swift]: I get the reason, but it will kick back one day...

**Comment by bonkey** [comment]
👍

File: AtlasUI/AtlasUI/LoaderView.swift (hunk 1 above)
Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/LoaderView.swift]: what about moving it to usermessage?

**Pull request merged**